### PR TITLE
Move to yarn workspaces

### PR DESCRIPTION
**Pull request by gsgalloway**

### CODE REVIEW

File: .github/workflows/farcaster-js.yml [hunk 1]
```diff
@@ -18,30 +18,33 @@ jobs:
...
-        run: yarn test
-        env:
-          INTEGRATION_TEST_USER_MNEMONIC: ${{ secrets.INTEGRATION_TEST_USER_MNEMONIC }}
-
-      - name: Release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: npx semantic-release
+        run: yarn workspaces foreach --all run test
```
Comment (piratekev) [inline .github/workflows/farcaster-js.yml:44]: I think we still need the env vars here

Comment (piratekev) [reply .github/workflows/farcaster-js.yml:44]: I'll need them for the neynar tests, it's fine for now

File: packages/farcaster-js-hub-rest/package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "dist/**",
+    "types/**"
+  ],
+  "scripts": {
+    "clean": "yarn clean:dist && yarn clean:generated",
+    "clean:dist": "rm -rf dist",
+    "clean:generated": "rm -rf src/openapi/generated",
+    "cz": "yarn run -T cz",
+    "generate": "yarn run generate:openapi",
+    "generate:docs": "md-magic --path './README.md' && typedoc --plugin typedoc-plugin-markdown",
```
Comment (piratekev) [inline packages/farcaster-js-hub-rest/package.json:49]: do these colon commands work inside a workspace?

**Review by piratekev — APPROVED** [review]

File: packages/farcaster-js-hub-rest/package.json (hunk 2 above)
Comment (gsgalloway) [reply packages/farcaster-js-hub-rest/package.json:49]: Seems so! I've tried:

```sh
yarn workspace @standard-crypto/farcaster-js-hub-rest run generate:docs
```

```sh
cd packages/farcaster-js-hub-rest/
yarn run generate:docs
```

**Pull request merged**